opentrons-ai-client/src/resources/hooks/useGetAccessToken.ts

```diff
@@ -1,22 +1,39 @@
 import { useAuth0 } from '@auth0/auth0-react'
-import { PROD_AUTH0_AUDIENCE, STAGING_AUTH0_AUDIENCE } from '../constants'
+import {
+  PROD_AUTH0_AUDIENCE,
+  STAGING_AUTH0_AUDIENCE,
+  LOCAL_AUTH0_AUDIENCE,
```

Comment:
we use VSCode to sort import statements `shift + cmd + p`


https://github.com/user-attachments/assets/089a3fd4-d1a1-4e2c-8312-8775988abe17



```suggestion
  LOCAL_AUTH0_AUDIENCE,
  PROD_AUTH0_AUDIENCE,
  STAGING_AUTH0_AUDIENCE,
```

Reply:
Many thanks Koji!